.github/workflows/sync.yml

```
@@ -27,7 +27,7 @@ jobs:
           else
             curl -O https://nf-co.re/pipeline_names.json
           fi
-          echo "name=matrix::$(cat pipeline_names.json)" >> $GITHUB_OUTPUT
+          echo "matrix==$(cat pipeline_names.json)" >> $GITHUB_OUTPUT
```

```suggestion
          echo "matrix=$(cat pipeline_names.json)" >> $GITHUB_OUTPUT
```
Not sure we want a comparison here, or do we?